cmd/shield/commands/backends/list.go

```diff
@@ -46,13 +76,116 @@ func cliListBackends(opts *commands.Options, args ...string) error {
 		}
 
 		if isCurrent {
-			backend.Name = ansi.Sprintf("@G{%s}", backend.Name)
-			backend.Address = ansi.Sprintf("@G{%s}", backend.Address)
+			backend.Name = greenify(backend.Name)
 		}
 
 		t.Row(backend, backend.Name, backend.Address)
 	}
-	t.Output(os.Stdout)
 
-	return nil
+	return &t
+}
+
+func verboseList(backends []*api.Backend) (*tui.Table, error) {
+	t := tui.NewTable("Name", "Backend URI", "Insecure", "Token", "CA Cert")
+
+	for _, backend := range backends {
+		isCurrent := config.Current() != nil && backend.Name == config.Current().Name
+
+		isInsecure := fmt.Sprintf("%t", backend.SkipSSLValidation)
+
+		if backend.CACert != "" {
+			var err error
+			backend.CACert, err = certInfoString(backend.CACert)
+			if err != nil {
+				return nil, err
+			}
+		}
+
+		if isCurrent {
+			backend.Name = greenify(backend.Name)
+		}
+
+		if backend.SkipSSLValidation {
+			isInsecure = redify(isInsecure)
+		}
+
+		t.Row(backend, backend.Name, backend.Address, isInsecure, backend.Token, backend.CACert)
+	}
+
+	return &t, nil
+}
+
+func greenify(s string) string {
+	return ansi.Sprintf("@G{%s}", s)
+}
+
+func redify(s string) string {
+	return ansi.Sprintf("@R{%s}", s)
+}
```

Comment:
No.  Just no.

Remove the greenify and redify functions, and use the inline calls to `ansi.Sprintf`